--- sysvinit-2.86/src/killall5.c.orig	2010-05-26 13:43:02.000000000 +0200
+++ sysvinit-2.86/src/killall5.c	2010-05-26 13:44:52.000000000 +0200
@@ -621,7 +621,7 @@
 	sid = (int)getsid(0);
 	pid = (int)getpid();
 	for (p = plist; p; p = p->next)
-		if (p->pid != pid && p->sid != sid && !p->kernel)
+		if (p->pid != pid && p->sid != sid && !p->kernel && p->pid != 1)
 			kill(p->pid, sig);
 
 	/* And let them continue. */
